klimaat_helpdesk/templates/base.html

```diff
@@ -26,7 +26,7 @@
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments);}
   gtag('js', new Date());
-  
+
   gtag('config', 'G-EZ555D7QSW');
```

Comment:
Are you sure the ``gtag('config', 'G-EZ555D7QSW');`` line is needed? Wouldn't that defeat the purpose of anonymizing ip? (Not sure, just checking)

Reply:
not sure to be honest. I think there are many other config parameters than ip. the second line undo what is set in the config. I tried to find how to change it in the control panel but it is a complete zoo.